### DISCUSSION TITLE
Latest ASP.NET Core GraphQL Boxed is missing few features like AddCorrelationIdFluent() method in the Startup.cs class

### DISCUSSION THREAD

**Discussion by santoshpatro** [Dotnet-Boxed/Templates#481] [Q&A]
### Question
I have installed the latest version of ASP.NET Core GraphQL Boxed with  ASP.NET Core 3.1. On creating a sample project I found that the latest Startup.cs file is missing few methods like AddCorrelationIdFluent() in the ConfigureServices() method of Startup.cs

Can you please help me to know does the latest template implicitly manages this feature or  do I have to add any custom code for it ?

It would be very helpful if you can help me to get the list of features that were available in the last version of ASP.NET Core GraphQL Boxed template for my further analysis. 

### Project template

ASP.NET Core GraphQL Boxed with  ASP.NET Core 3.1

**Comment by RehanSaeed** [comment]
This is all documented in the releases page:

https://github.com/Dotnet-Boxed/Templates/releases
